woocommerce/woocommerce-android
# [Customer Search] Add loading indicator when a customer selected and we fetch more data

Closes: #9394


### Description

Add partial loading indicator when we load customer info after it was clicked

### Testing instructions

* Start order creation
* Add customer info
* Find registered customer and click on it
* Click on it
* Notice partial loading indicator at the top

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/bf38236a-f8af-47cf-9393-20f376d68bcb
